src/libs/feature-previews-config.ts

```diff
@@ -97,12 +97,13 @@ const featurePreviewsConfig: readonly FeaturePreview[] = [
   },
   {
     id: FIRECLOUD_UI_MIGRATION,
-    title: 'Firecloud UI Feature Migration',
-    description: 'Enabling this feature will update replaceable links to Firecloud UI with new links to Terra UI',
-    feedbackUrl: `mailto:dsp-workflow-management@broadinstitute.org?subject=${encodeURIComponent(
-      'Feedback on deprecating Firecloud UI'
+    title: 'Terra Workflow Repository Improvements',
+    description:
+      "Enabling this feature will gives users access to the new Terra Workflow Repository UI. This replaces the Broad Methods Repository UI (and doesn't affect any workflows previously created).",
```

Comment:
I attempted to emphasize what had been missing in Terra (create & edit) and try to explain that these are two UIs viewing the same backend data.
```suggestion
      "Create and edit workflows with the built-in Terra Workflow Repository UI. Changes made in Terra are reflected in the external Broad Methods Repository, and vice-versa.",
```

Reply:
What do you think about this? This also conveys that Broad Methods Repository is being replaced. 🤔 
```
Enabling this feature will allow creating and editing workflows with the built-in Terra Workflow Repository UI. This replaces the external 
Broad Methods Repository. But changes made in Terra are reflected in the external Broad Methods Repository, and vice-versa.
```

Reply:
Seems good to me!